Source/ACE.Server/Command/Handlers/AdminCommands.cs

```diff
@@ -1084,6 +1084,60 @@ public static void HandleTeleportLOC(Session session, params string[] parameters
             }
         }
 
+        // teleplayerloc <player name> cell x y z [qx qy qz qw]
+        [CommandHandler("teleplayerloc", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 4,
+            "Teleports a player to the specified location.",
+            "cell [x y z] (qw qx qy qz)\n" +
+            "@teleplayerloc follows the same number order as displayed from @loc output\n" +
+            "Example: @teleplayerloc <player name> 0x7F0401AD [12.319900 -28.482000 0.005000] -0.338946 0.000000 0.000000 -0.940806\n" +
+            "Example: @teleplayerloc <player name> 0x7F0401AD 12.319900 -28.482000 0.005000 -0.338946 0.000000 0.000000 -0.940806\n" +
+            "Example: @teleplayerloc <player name> 7F0401AD 12.319900 - 28.482000 0.005000")]
+        public static void HandleTeleportPlayerLOC(Session session, params string[] parameters)
+        {
+            var playerName = string.Join(" ", parameters[0]);
```

Comment:
this will not work for players w/ spaces in their names

should be just parameters, without the [0]